<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waitstu2</class>
 <widget class="QWidget" name="waitstu2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btn_Setting">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_shutdown">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_gif">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
